modules/exploits/multi/http/clipbucket_fileupload_exec.rb

```diff
@@ -0,0 +1,140 @@
+##
+# This module requires Metasploit: https://metasploit.com/download Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "ClipBucket Unauthenticated Arbitrary File Upload lower than 4.0.0 - Release 4902",
```

Comment:
Usually the version is kept out of the module name, in favor of describing vulnerable functionality:

Something like:
* `ClipBucket Unauthenticated Arbitrary File Upload`; or
* `ClipBucket beats_uploader Unauthenticated Arbitrary File Upload`